src/styles/base/_base.scss

```diff
@@ -0,0 +1,34 @@
+@use 'styles/abstracts' as *;
+
+*,
+*::after,
+*::before {
+  margin: 0;
+  padding: 0;
+  box-sizing: inherit;
+}
+
+html {
+  // This defines what 1rem is
+  font-size: 62.5%; //1 rem = 10px; 10px/16px = 62.5%
```

Comment:
Why are we setting default 1 rem as 10px? By default most browsers set it to 16px and I think that should be enough for this project as well.

Reply:
@hhu-wtag bhai, I have made a modification where `1rem` is equal to `10px`. This change allows for easier conversion from px to rem units. For example, if we want to convert `37px` to rem, we can simply `divide 37 by 10`, resulting in `3.7rem`. Where `37/16 = 2.3125` which may be irritating sometimes. This adjustment has helped simplify the calculation process for me. Anyways, bhai, I'm not sure if it's considered good practice or not. Please let me know your thoughts on this. thanks in advance

Reply:
By setting the font size of html element to 62.5% you are setting the default font to ~10px. Setting font size to this low has some problems. 
1. The original design was made using 16px as default font size.
2. People with eyesight problems will face difficulty and have to zoom in to navigate the site. 

Please keep this in mind and think about the solution.

Reply:
Hi @ray-wtag bhai, this is an old trick known as `The 62.5% Font Size Trick` [introduced](https://clagnut.com/blog/348/) by Richard Rutter in 2004. Here are some references I have used before:
https://www.toheeb.com/en/62.5-font-size-trick/
https://www.toheeb.com/en/62.5-font-size-trick/
https://javascript.plainenglish.io/why-i-change-the-font-size-to-62-5-in-every-project-45c5ff785fb5

I have found some controversy with this trick. So have dropped it. Thanks